tool/planet/etcd.go

```diff
@@ -163,9 +154,34 @@ func etcdBackup(backupFile string, backupPrefix []string) (err error) {
 	return nil
 }
 
+func etcdMigrate(fromVersion, toVersion string) (err error) {
+	if fromVersion == "" {
+		fromVersion, _, err = readEtcdVersion(DefaultEtcdCurrentVersionFile)
+		if err != nil {
+			if !trace.IsNotFound(err) {
+				return trace.Wrap(err)
+			}
+			fromVersion = AssumeEtcdVersion
```

Comment:
Do we really need to assume anything anymore? I think all versions we're upgrading from (5.5) now should have the "etcd version" file?

Reply:
I would set a minimum version of 5.5 where this is a fact - removing the assume case will definitely simplify this.